worker/worker.py

```diff
@@ -76,6 +76,7 @@ def setup_config_file(config_file):
         ("parameters", "min_threads", "1"),
         ("parameters", "fleet", "False"),
         ("parameters", "use_all_cores", "False"),
+        ("parameters", "write_config", "False"),
```

Comment:
It' seems to work commenting out these two lines
`# ("parameters", "write_config", "False"),`

Reply:
@ppigazzini Thanks, I overlooked step 2. Should have figured that out myself :-/